assignments/objects.js

```diff
@@ -18,27 +19,61 @@ const example = {
 }
 
 // Write your intern objects here:
-
+const mitzi = {
```

Comment:
You have this written as separate objects, could you think of another way where these objects are nested into one??